.github/workflows/github-ci.yml

```diff
@@ -250,23 +259,26 @@ jobs:
 
   build_ionosphereTests:
     # Build IonosphereSolverTests miniApp (currently broken)
-    runs-on: ubuntu-latest
-    container: ursg/vlasiator_ci:20230220_1
+    runs-on: carrington
+      #container: ursg/vlasiator_ci:20230220_1
     needs: build_libraries
     steps:
     - name: Checkout source
       uses: actions/checkout@v3 
-    - name: Download libraries
```

Comment:
here as well the libraries should be downloaded ( and I guess we could return it to the cloud)